SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Utils/Misc/SpatialActivationFlags.cpp

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "CoreMinimal.h"
+
+#include "TestDefinitions.h"
+#include "Tests/AutomationCommon.h"
+#include "Runtime/EngineSettings/Public/EngineSettings.h"
+
+static const bool s_earliestFlag =  GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
+
+FString const s_earliestFlagReport = "Spatial activation Flag [Earliest]:";
```

Comment:
Please capitalize the first letter of variables. And the only hungarian notation we use is for booleans.

Reply:
Please wrap all strings in the TEXT() macro